doc/developer/debugging.rst

```diff
@@ -57,4 +61,35 @@ account:
 
     sudo /etc/init.d/postgres reload
 
-Logs are available in the ``/var/log/postgresql/postgresql-9.0-main.log`` file.
+Logs are available in the ``/var/log/postgresql/postgresql-9.*-main.log`` file.
+
+Performance or network error
+----------------------------
+
+For performance and proxy issue be sure that add the internal URL in the config file
+are on ``localhost`` (use ``curl "http://localhost/<path>" -H Host:<server_name>``
+to test it).
+
+Tilecloud chain
+...............
+
+Points to check with TileCloud chain::
+
+ * Be sure if that metatile are desable it's for good reason.
+ * That the ``empty_metatile_detection`` and ``empty_tile_detection`` are configured correctly.
```

Comment:
Make sure that``empty_metatile_detection`` and ``empty_tile_detection`` are configured correctly.